ateliers/snake/bonus.py

```diff
@@ -0,0 +1,171 @@
+########################################
+###          Objectifs bonus         ###
+###      (c) Zeste de Savoir (c)     ###
+###           Licence GPL            ###
+###         Auteur : TAlone          ###
+########################################
+
+# Quelques bonus pour les participants en avance :
+# (bonus 1 - aisé) - empêche le serpent d'aller en arrière ;
+# (bonus 2 - moyen) - ajout d'un compte de points ;
+# (bonus 3 - difficile) - ajout d'un écran pour rejouer ;
+# (bonus 4 - difficile) - ajout de coins au serpent.
+
+# Import de la bibliothèque du coding gouter (fonctions d'abstraction)
+from bibliotheque import *
+
+# Fonction principale d'initialisation
+def initialisation(jeu):
+	# Création d'un serpent à l'écran
+	jeu.serpent = Serpent()
+
+	# Création d'une pomme
+	jeu.pomme = jeu.position_aleatoire_pomme()
+
+	# (B2) Déclaration d'un compteur de points
+	jeu.points = 0
+
+	# (B3) Création d'une variable de perte
+	jeu.serpent.vivant = True
+	# (B3) Ajout du texte pour relancer la partie
+	jeu.ajouter_texte("NVPartie", "Pour rejouer, appuyez sur ESPACE")
```

Comment:
```suggestion
	jeu.ajouter_texte("Nouvelle Partie", "Pour rejouer, appuyez sur ESPACE")
```

C'est une chaîne de caractère donc autant éviter les abréviations, non ? :)